### ISSUE TITLE
Modid dimstorage fires events every second and saturates console.

### ISSUE THREAD

**Issue by fffaria** [Edivad99/DimStorage#30]
**Describe the bug**
I have no idea why, but ever since the first time I have installed this mod in our server, we are getting these same messages in less than a second in console:
``` 
[16Jun2020 02:42:38.942] [Thread-1/DEBUG] [net.minecraftforge.fml.config.ConfigFileTypeHandler/CONFIG]: Config file dimstorage-common.toml changed, sending notifies
[16Jun2020 02:42:38.945] [Thread-1/DEBUG] [net.minecraftforge.fml.config.ConfigFileTypeHandler/CONFIG]: Config file forge-server.toml changed, sending notifies
[16Jun2020 02:42:38.945] [Thread-1/FATAL] [net.minecraftforge.common.ForgeConfig/CORE]: Forge config just got changed on the file system!  
```
The mod works well, however. It's just the console that gets filled all the time with the same message every second. 

**Configuration**
Mod version: DimStorage-1.14.4-2.4.5
Forge version: 28.2.0
Minecraft version: 1.14.4
Are you playing on server?: Yes.

**To Reproduce**
Upload the mod using the version above.

**Expected behavior**
No fatal or log messages at every second

If anything, I got the log to help with searching the problem! 

Thank you for the time taken to read all this!

**Comment by Edivad99** [comment]
Please give me the full log. This event is raised when you launch the `/dimstorage` commands. Spam is normal if all players continue to run this command

**Comment by fffaria** [comment]
Here is the log after the server restart @Edivad99
[debug.log](https://github.com/Edivad99/DimStorage/files/4806606/debug.log)

**Comment by Edivad99** [comment]
Ok, can you give me the files `dimstorage-common.toml` and `forge-server.toml`?

**Comment by Edivad99** [comment]
However in theory I think I understand where the problem is, but it will be solved in mid-July.

**Comment by fffaria** [comment]
Here you go! It's okay! I am glad that you guys understand the problem!
[srv.zip](https://github.com/Edivad99/DimStorage/files/4808512/srv.zip)

**Comment by Edivad99** [comment]
Ok, I have done other investigations, have you tried to see if this problem also occurs in 1.15.2? I ask why Forge will soon stop offering support for 1.14.4 and therefore probably this problem will never be solved as I have new features that will be implemented only in 1.15.2 and 1.16.
So I would ask you to convert your modpack to 1.15.2 and check for the same problem

**Comment by fffaria** [comment]
Hello there! I shall update the server for the version 1.16 later on and I am going to check with you if it's good! 

Edit: I have installed the mod in the latest version (1.16.1), and the warnings stopped. Thanks!
